lib/infrastructure/templates/renderTemplates/sub-service-request-approved/email/html.ejs

```diff
@@ -255,17 +255,17 @@
                     </div>
                 </dl>
           
-
-                <p>To access this sub-service, sign in to your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
-
+                <% if (isMultipleSubServicesReq) { %>
+                    <p>To access your sub-services, sign in to your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
+                <% } else { %>
+                    <p>To access your sub-service, sign in to your DfE Sign-in account at <a href="<%=signInUrl%>">services.sign-in</a>. Or, you can select the button below to sign in to your account.</p>
+                <% } %>
                 <p>
-                <a class="button-link" href="<%=signInUrl%>">Go to your DfE Sign-in account</a>
+                <a class="button-link" href="<%=signInUrl%>">Go to DfE Sign-in account</a>
                 </p>
-                <p>Once signed in, select the service this sub-service applies to from the list on your services page.</p>
                 <p>
                     Kind regards,
```

Comment:
As per the prototype, there is no comma after Kind regards, please double check with the UI/UX  designer 
![image](https://github.com/DFE-Digital/login.dfe.notifications.jobs/assets/70764046/2f43a6bd-f56e-4dd6-93df-9d1af0e72d2a)

Reply:
removed as per design fixed next checkin